server/athenian/api/precompute/accounts.py

```diff
@@ -483,6 +484,33 @@ async def _ensure_root_team(account: int, sdb: Database) -> int:
     return await sdb.execute(insert(Team).values(team.create_defaults().explode()))
 
 
+async def _sync_bots_team_members(
+    account: int,
+    local_bots: Collection[str],
+    prefixer: Prefixer,
+    sdb: Database,
+    log: logging.Logger,
+) -> None:
+    """Update the members of the bots team if needed.
+
+    Missing bots are added to the Bots team in sdb.
+    Existing bots in the Bots team are never removed.
+    """
+    bot_ids = list(
+        set(chain.from_iterable(prefixer.user_login_to_node.get(u, ()) for u in local_bots)),
+    )
+
+    async with sdb.connection() as sdb_conn:
+        async with sdb_conn.transaction():
+            bot_team = await get_bots_team(account, sdb)
+            if bot_team is None:
+                return
+            log.info("syncing bots team members")
+            new_members = await sync_team_members(bot_team, bot_ids, sdb_conn)
+            if new_members:
+                log.info("added members to Bots team: %s", ",".join(map(str, new_members)))
```

Comment:
The flow should be:

- Load the current bots team members.
- Find the new bots team members minus "extra". This is exactly the code inside `_ensure_bot_team` down until we insert into the DB. We factor that code out.
- If the members are the same, return. Otherwise, issue a SQL update.

According to how I see it, all the old code lives as-is. Also, the new code becomes simpler.